lib/rspec/core/configuration.rb

```diff
@@ -1412,13 +1415,32 @@ def self.delegate_to_ordering_manager(*methods)
       #     end
       #   end
       #
-      #   describe MyClass, :order => :reverse do
+      #   RSpec.describe 'MyClass', :order => :reverse do
       #     # ...
       #   end
       #
       # @note Pass the symbol `:global` to set the ordering strategy that
       #   will be used to order the top-level example groups and any example
       #   groups that do not have declared `:order` metadata.
+      #
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.register_ordering :global do |examples|
+      #       acceptance, other = examples.paritition do |example|
+      #         example.metadata[:type] == :acceptance }
+      #       end
+      #       [rest, acceptance]
```

Comment:
As discussed by @e2 in #2247 it does not work to return a list-of-lists.  The RSpec ordering API calls your block with a flat list of groups or examples and expects you to return a flat list of groups or examples.  If you do `[rest, acceptance]` it leads to exceptions.  This should be: `rest + acceptance` (or something similar).